Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM alpine/helm:3.13.3
+FROM alpine/helm:3.14.4
 
 RUN apk update && \
     apk add --no-cache aws-cli yq
 
 RUN apk upgrade --no-cache
 
-ARG SKAFFOLD_VERSION=v2.11.0
+ARG SKAFFOLD_VERSION=v2.12.0
```

Comment:
I missed to mention that the Dockerfile change requires you to release a new image version, bumping the Action's image tag.